shared/chat/conversations-list/index.desktop.js

```diff
@@ -1,62 +1,130 @@
 // @flow
 import React from 'react'
-import {Box, Text, Avatar, Icon, Usernames} from '../../common-adapters'
+import {Box, Text, MultiAvatar, Icon, Usernames} from '../../common-adapters'
 import {globalStyles, globalColors} from '../../styles'
 import {participantFilter} from '../../constants/chat'
 import {formatTimeForConversationList} from '../../util/timestamp'
 
 import type {Props} from './'
+import type {InboxState} from '../../constants/chat'
 
-const ConversationList = ({inbox, onSelectConversation, selectedConversation, onNewChat, nowOverride}: Props) => (
-  <Box style={{...globalStyles.flexBoxColumn, backgroundColor: globalColors.darkBlue4, width: 240}}>
+const AddNewRow = ({onNewChat}: Props) => (
+  <Box
+    style={{...globalStyles.flexBoxRow, ...globalStyles.clickable, minHeight: 48, justifyContent: 'center', alignItems: 'center', flexShrink: 0}}
+    onClick={() => onNewChat()}>
+    <Icon type='iconfont-new' style={{color: globalColors.blue, marginRight: 9}} />
+    <Text type='BodyBigLink'>New chat</Text>
+  </Box>
+)
+
+const rowBorderColor = (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) => {
```

Comment:
In the multi avatar case we actually have a blue border around the second avatar. If there is an unread then we orange circle the last avatar (can only be one also)